tests/nordic_softdevice/tests/01-run.py

```diff
@@ -1,14 +1,17 @@
 #!/usr/bin/env python3
 
 import sys
+import time
 from testrunner import run
 
 
 def testfunc(child):
-    child.expect("All up, running the shell now")
+    child.expect_exact("All up, running the shell now")
```

Comment:
What is the reasoning behind replacing this? The result should be the same.

Reply:
This is part of the cleanup of this PR.

Reply:
Not sure what you mean by that. Why is `expect_exact()` cleaner than `expect()`?

Reply:
`expect()` takes a pattern as argument. From the doc, `expect_exact()` is similar to `expect()`, but uses plain string matching.
I admit the cleanup is a bit nitpicky in this case but it doesn't hurt.

Reply:
Ok :-)